stripe/src/test/java/com/stripe/android/paymentsheet/PaymentSheetPaymentMethodsListFragmentTest.kt

```diff
@@ -52,7 +52,7 @@ class PaymentSheetPaymentMethodsListFragmentTest {
         val scenario = launchFragmentInContainer<PaymentSheetPaymentMethodsListFragment>()
         scenario.onFragment {
             val recycler = recyclerView(it)
-            assertThat(recycler.adapter).isNull()
+            assertThat(recycler.adapter).isNotNull()
```

Comment:
nit: maybe change this to `assertThat(recycler.adapter).isEmpty()` or whatever that needs to be.